understanding/20/sensory-characteristics.html

```diff
@@ -21,8 +21,8 @@ <h2>Intent of Sensory Characteristics</h2>
          to the right"). 
          Some users with disabilities are not able to perceive shape or position due to the
          nature of the assistive technologies they use. 
-         This Success Criterion requires that additional information be provided to clarify
-         anything that is dependent on this kind of information.
+         This Success Criterion requires that additional information be provided as part of
+         the instructions to clarify anything that is dependent on this kind of information.
```

Comment:
SUggest: This Success Criterion requires that additional information be provided to clarify	  instructions that are dependent on this kind of information.

Reply:
yup, a bit shorter. i can go with that...